build.gradle

```diff
@@ -318,16 +344,38 @@ project(':core') {
     testCompile.exclude module: 'api-ldap-schema-data'
   }
 
-  tasks.create(name: "copyDependantLibs", type: Copy) {
-    from (configurations.testRuntime) {
-      include('slf4j-log4j12*')
-    }
+  tasks.create(name: "copyDependantLibsScala", type: Copy) {
     from (configurations.runtime) {
-      exclude('kafka-clients*')
+      include('*scala*.jar')
     }
-    into "$buildDir/dependant-libs-${versions.scala}"
+    into "$baseBuildDir/dependant-libs-scala-${versions.scala}"
   }
+  tasks.create(name: "copyTestDependantLibsScala", type: Copy) {
+    from (configurations.testRuntime - configurations.runtime) {
+      include('*scala*.jar')
+    }
+    into "$baseBuildDir/test-dependant-libs-scala-${versions.scala}"
+  }
+
+  task linkJars(dependsOn: ['copyDependantLibsJava', 'copyDependantLibsScala', 'copyTestDependantLibsJava', 'copyTestDependantLibsScala']) {
```

Comment:
Can linkJars be broken into 2 tasks (linkJars and linkScalaJars). That way linkScalaJars can stay here, but linkJars can be moved up to subprojects and de-duplicated like copyDependantLibsJava was.